improver/precipitation_type/snow_splitter.py

```diff
@@ -63,7 +64,7 @@ def separate_input_cubes(cubes: CubeList) -> Tuple[Cube, Cube, Cube]:
 
         return (rain_cube, snow_cube, precip_cube)
 
-    def process(self, cubes: CubeList,) -> Cube:
+    def process(self, *cubes: Union[Cube, CubeList]) -> Cube:
```

Comment:
Same question about typing, here and anywhere else.

Reply:
we need docs here too :)

Reply:
as above, there is docs here already